common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -1987,23 +1987,6 @@ function update_schematics() {
 }
 window.update_schematics = update_schematics;
 
-// add ourselves to the tasks that get performed when window is loaded
-function add_schematic_handler(other_onload) {
```

Comment:
The only references to this function are in the two wiki templates, removed in this commit.